util/src/main/java/com/coderanch/util/matchers/Matchers.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.coderanch.util.matchers;
+
+import java.util.function.*;
+
+import org.hamcrest.CustomTypeSafeMatcher;
+import org.hamcrest.Matcher;
+
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.isEmptyString;
+import static org.hamcrest.Matchers.not;
+
+/**
+ * Utility class to create common matchers.
+ */
+public final class Matchers {
+
+    private Matchers() {}
+
+    private static final Matcher<String> TRIMMED_STRING_MATCHER = createTypeSafeMatcher(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this variable to comply with Java Code Conventions. ([squid:S1213](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS1213))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack-parent&pullRequest=11&issues=AWaB526VksgJibV5mvF_&open=AWaB526VksgJibV5mvF_)

[ISSUE_KEY]: AWaB526VksgJibV5mvF_